### ISSUE TITLE
A11y: [Send an image] 'i' icon button next to placeholder has no alterna...

### ISSUE THREAD

**Issue by mxandreas** [element-hq/element-x-android#6379]
- **Checkpoint ID:** 11.1.1.1a/9.1
- **Component:** Send an image

### Issue Description
'i' icon button next to placeholder has no alternative text.

### Recommendation
Add descriptive alt text to the information button.

--- 
*Report Date: 25.02.2026*
